analyzer/codechecker_analyzer/analysis_manager.py

```diff
@@ -335,6 +335,8 @@ def handle_failure(source_analyzer, rh, zip_file, result_base, actions_map):
 
     LOG.debug("ZIP file written at '%s'", zip_file)
 
+    rh.postprocess_result()
```

Comment:
Maybe it would be good to leave a comment here which will remember us why do we need to call this function on analysis failure too.